### ISSUE TITLE
card details dialog on desktop is annoying

### ISSUE THREAD

**Issue by bgmnt** [nextcloud/deck#3339]
title is no complain, just a description. i love deck ;)
when creating a new card and i only need a title and want to throw it in quickly (to-do-list), this dialog on the right opens.

- it blocks the view on the other lists (bad on smaller screens/browser windows)
- it won't go away without searching the closing button - escape button or clicking 
- somewhere does not work
- it even appears when i just move a card and do not even want to edit something.

for me this is really a pain in usability.
possible solutions:

- do not call dialog when clicking a card only once
- put an edit function in the context menu of a card
- do not call dialog when moving cards
- close dialog when clicking elsewhere (like new cards disappear when you don't save them, but the content is still there)

thx and greetings!

**Comment by stefan-niedermann** [comment]
You currently also have the option to switch from the "Details in right sidebar" mode to "Details in modal dialog" mode:

![grafik](https://user-images.githubusercontent.com/4741199/134816839-475940d2-ea74-444b-8457-af9bf218ce80.png)

Current implementation is:
- Sidebar mode: Opens directly after entering a new card title
- Dialog mode: Does **not** open after entering a new card title

Funny thing: Someone else requested (https://github.com/nextcloud/deck/issues/2896) to change the behavior of the dialog mode to also match the sidebar mode (directly open after editing) - well it depends on how one is using Deck.

There is also a (in my opinion) great proposal in (https://github.com/nextcloud/deck/issues/2896#issuecomment-805885334) to differentiate the behavior based on whether the user pressed <kbd>Enter</kbd> or <kbd>Ctrl</kbd> + <kbd>Enter</kbd>. Since this proposal would cover also your needs as far as i can see, i suggest to close this issue in favor of #2896.

**Comment by bgmnt** [comment]
thank you @stefan-niedermann 
i think i can live with control+enter - good workaround. the dialog mode feels more natural, i'll use this.

**Comment by bgmnt** [comment]
nope, pardon, one issue remains: even if i only move a card around on the board, the details dialog opens. that makes no sense - dragging items should not lead to any other action (and does not in other task boards).

**Comment by juliushaertl** [comment]
This is actually a duplicate of https://github.com/nextcloud/deck/issues/2594 so I'm closing this one in favor of the other.
